containers/Dockerfile.intel-oneapi-ufs-s2s-coupled-dev

```diff
@@ -0,0 +1,35 @@
+FROM intel-oneapi-hpc-stack:ubuntu20
+
+SHELL ["/bin/bash", "-c"]
+
+RUN apt-get update -y && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends -o=Dpkg::Use-Pty=0 \
+        csh \
```

Comment:
Do we need all these shells? `csh`, `ksh`, `tcsh`?
Also, why have `tex, lynx, nedit, emacs`? Is `vi` not enough?
We are not building documentation, so `doxygen` is not necessary either.

Reply:
These extra packages were included based on a jcsda container found here (https://github.com/JCSDA-internal/containers/blob/develop/Dockerfile.intel19-impi-dev).  Since I don't have much experience with how these dev containers will be used, I put in more shell options as I guessed that researchers might want them to be available.  I will remove them as they are not critical to the ufs-s2s-model's functionality.